ethcore/src/miner/filter_options.rs

```diff
@@ -0,0 +1,823 @@
+use ethereum_types::{Address, U256};
+use serde::de::{Deserialize, Deserializer, Error, MapAccess, Visitor};
+use std::fmt;
+use std::marker::PhantomData;
+
+/// This structure provides filtering options for the pending transactions RPC API call
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct FilterOptions {
+    /// Contains the operator to filter the sender value of the transaction
+    pub sender: FilterOperator<Address>,
+    /// Contains the operator to filter the receiver value of the transaction
+    pub receiver: FilterOperator<Address>,
```

Comment:
Could we do `FilterOperator<Option<Address>>`? That wouldn't require the special case for `FilterOperator` as it is currently.

Reply:
Could we also use `from/to` instead of `sender/receiver`? I think it's more inline with other RPC structs.

Reply:
Yes, this can simplify certain aspects. I will implement this.